### DISCUSSION TITLE
Disabling translations?

### DISCUSSION THREAD

**Discussion by Tjuh** [smplayer-dev/smplayer#63] [Q&A]
Is there any way to disable the creation of translation files during compilation? Only need/use the English language.

**Comment by smplayer-dev** [comment]
Remove this line from the Makefile: `cd src && $(LRELEASE) smplayer.pro`

**Comment by Tjuh** [comment]
Had to remove these lines as well `-rm src/translations/smplayer_*.qm` `install -m 644 src/translations/*.qm $(DESTDIR)$(TRANSLATION_PATH)` but you put me on the right track. Thanks a bunch mate, most appreciated.
